tests/negative/security/tls.go

```diff
@@ -34,10 +34,18 @@ func init() {
 	config := &tls.Config{Certificates: []tls.Certificate{cer}}
 	customCAServer.TLS = config
 	customCAServer.Config.ErrorLog = log.New(ioutil.Discard, "", 0)
+
+	cer1, err := tls.X509KeyPair(publicKey1, privateKey1)
+	if err != nil {
+		panic(fmt.Sprintf("error loading x509 keypair: %v", err))
+	}
+	config1 := &tls.Config{Certificates: []tls.Certificate{cer1}}
+	customCAServer1.TLS = config1
 	customCAServer.StartTLS()
```

Comment:
We're only calling `StartTLS()` on `customCAServer`, not `customCAServer1`.  I see that some tests also call `StartTLS()`.  Should we be calling both here, or neither?